compute_runtime/compute_runtime.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 Fastly, Inc.
+//
+package compute_runtime
+
+import (
+  "time"
+
+  "github.com/fastly/compute-sdk-go/internal/abi/fastly"
+)
+
+func GetVCPUTime() (time.Duration, error) {
+  milliseconds, err := fastly.GetVCPUMilliseconds()
```

Comment:
The check failures here are because you will also need to add the hostcalls to `hostcalls_noguest.go`, which is what gets compiled when the `go:build` conditions from your new `compute_runtime_guest.go` file don't apply.  It's fine to just have them return a not-implemented error like the other functions in that file.

Reply:
Thank you for the pointer! Would've taken me forever to find this. Done now.